Makefile

```diff
@@ -0,0 +1,2 @@
+install:
+	pip install -e .
```

Comment:
`pip install --user -e .` should do that. `./setup.py develop` does a similar or maybe the same thing but I rarely use it unless a project is using distutils (yuck).

You can also just not install it at all, and run it directly like `cinderblock/commitstatus.py`.

But really, it sounds like you are developing the [testing ice cream cone](http://watirmelon.com/2012/01/31/introducing-the-software-testing-ice-cream-cone/). You have tox all set up but there are not unit tests. Add some unit tests, and then you won't be constantly needing to manually run the tool to test it.

I'd also suggest [entr](http://entrproject.org/) to constantly run the unit tests as you code. Some people use text editor plugins, too.